integration/fuse/src/main/java/alluxio/fuse/AlluxioFuseUtils.java

```diff
@@ -525,4 +527,27 @@ public static int call(Logger logger, FuseCallable callable, String methodName,
     }
     return ret;
   }
+
+  /**
+   * Resolves a FUSE path into {@link AlluxioURI} and possibly keeps it in the cache.
+   */
+  public static final class PathCacheLoader extends CacheLoader<String, AlluxioURI> {
```

Comment:
Is it just a cache or cache loader?

Not see any actual action?

Reply:
It's a cache loader, the actual cache is done in AlluxioJNI/JNRFuse

Reply:
but the whole cache can be shared as well

Reply:
Modified, PTAL, thanks